README.md

```diff
@@ -1,4 +1,7 @@
 [![Build Status](https://cloud.drone.io/api/badges/tinkerbell/hegel/status.svg)](https://cloud.drone.io/tinkerbell/hegel)
+![](https://img.shields.io/badge/Stability-Experimental-red.svg)
+
+This repository is [Experimental](https://github.com/packethost/standards/blob/master/experimental-statement.md) meaning that it's based on untested ideas or techniques and not yet established or finalized or involves a radically new and innovative style! This means that support is best effort (at best!) and we strongly encourage you to NOT use this in production.
```

Comment:
The Tinkerbell community should have their own patterns for this, which can certainly be modeled on the Packet model. I imagine it will quickly evolve into its own thing.

Put succinctly, we should put the relevant https://github.com/packethost/standards/ bits in https://github.com/tinkerbell/.github and link there within Tinkerbell projects.

Thoughts, @gianarb @rainleander  ?

Reply:
Definitely. But for the short term, this will work, eh?